test/e2e-common.sh

```diff
@@ -26,17 +26,13 @@
 source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/e2e-tests.sh
 
 function install_operator_resources() {
-  local platform=${2:-kubernetes}
-  echo ">> Deploying Tekton Operator Resources"
-  ko apply -f config/ || fail_test "Tekton Operator installation failed"
-
   # Make sure that everything is cleaned up in the current namespace.
   for res in tektonpipelines tektontriggers tektondashboards; do
     kubectl delete --ignore-not-found=true ${res}.operator.tekton.dev --all
   done
```

Comment:
shall we move this deleting to `make apply` on ln35. so that we can be sure that there are no CR instances on the cluster before we start the test.

Reply:
i think we can merge this rigth away, i shall rebase the e2e test pr and change it if it is better.